docs/modules/operation/pages/performance-data-collection/introduction.adoc

```diff
@@ -26,6 +26,6 @@ You may also want to change how _collectd_ works: when, how, and what data it co
 Learn how to manage performance data collection:
 
 * <<performance-data-collection/collectd/configuration.adoc#ga-collectd-configuration, collectd configuration>>
-* <<performance-data-collection/collectors/collectors.adoc#collectors-overview, collection for specific protocols>>
+* <<performance-data-collection/collectors.adoc#collectors-overview, collection for specific protocols>>
```

Comment:
Send right to reference section instead of stub to redirect.
```suggestion
* <<reference:configuration/performance-data-collection/introduction.adoc#ref-performance-data-collectors, collection for specific protocols>>
```